shared/components/data/themes-list-fetcher/index.jsx

```diff
@@ -1,44 +1,45 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	omit = require( 'lodash/object/omit' ),
-	once = require( 'lodash/function/once' ),
-	isEqual = require( 'lodash/lang/isEqual' );
+import React from 'react';
+import pick from 'lodash/object/pick';
+import omit from 'lodash/object/omit';
+import once from 'lodash/function/once';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-var ThemesStore = require( 'lib/themes/stores/themes' ),
-	ThemesListStore = require( 'lib/themes/stores/themes-list' ),
-	Actions = require( 'lib/themes/flux-actions' ),
-	Constants = require( 'lib/themes/constants' );
-
-function queryThemes( props ) {
-	Actions.query( {
-		search: props.search,
-		tier: props.tier,
-		page: 0,
-		perPage: Constants.PER_PAGE
-	} );
-
-	Actions.fetchNextPage( props.site );
-}
-
-function getThemesInList() {
-	return ThemesListStore.getThemesList().map( ThemesStore.getById );
-}
-
-function getThemesState() {
+import Constants from 'lib/themes/constants';
+import * as allActions from 'lib/themes/actions';
```

Comment:
Hmm, do we really want `allActions`? We could just `import { query, fetchNextPage } from 'lib/themes/actions'` and `connect()` to `bindActionCreators.bind( null, { query, fetchNextPage } )`, couldn't we?